### DISCUSSION TITLE
Plotting Beam Diagram

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#2279] [Ideas]
## Description

I want to plot this figure (written by [simplebendingpractice](https://github.com/alfredocarella/simplebendingpractice)) in 3D by pyvista :rocket:
![plot](https://user-images.githubusercontent.com/7513610/88742589-8e7e9900-d17d-11ea-86f1-1df24a9a3a9b.png)

like [this page](https://debug.pi.gr/default.aspx?ch=546)

## Example Data
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/alfredocarella/simplebendingpractice/master?filepath=simple_demo.ipynb)

**Comment by tkoyama010** [comment]
I think we could follow this.
https://blog.kitware.com/sketchalyze-structural-analysis-goes-mobile/

**Comment by adhikarig** [comment]
Hi @tkoyama010 
Have you got any luck with the plotting.

**Reply by tkoyama010** [reply]
This is something we are still looking to add. We think it can probably be achieved by using glyph.

**Comment by adhikarig** [comment]
Earlier, I used Matplotlib to plot these diagrams.  It was very slow to plot and sluggish.  I dont know about glyph.  I'm quite keen to learn about that.
